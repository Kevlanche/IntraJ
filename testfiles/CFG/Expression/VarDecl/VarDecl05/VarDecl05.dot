digraph G {
graph [splines=ortho, nodesep="1", ranksep="1"]
node [shape=rect, fontname=Arial];
{ rank = same; "Program[0]"}
{ rank = same; "List[1]"}
{ rank = same; "CompilationUnit[2]"}
{ rank = same; "List[3]"; "List[4]"}
{ rank = same; "ClassDecl[5]"}
{ rank = same; "Modifiers[6]"; "Opt[9]"; "List[10]"; "List[11]"}
{ rank = same; "List[7]"; "MethodDecl[12]"; "MethodDecl[47]"}
{ rank = same; "Modifier[8]"; "Entry[13]"; "Modifiers[14]"; "PrimitiveTypeAccess[16]"; "List[17]"; "List[18]"; "Opt[19]"; "Exit[20]"; "Entry[48]"; "Modifiers[49]"; "PrimitiveTypeAccess[51]"; "List[52]"; "List[61]"; "Opt[62]"; "Exit[63]"}
{ rank = same; "List[15]"; "Block[20]"; "List[50]"; "ParameterDeclaration[53]"; "ParameterDeclaration[57]"; "Block[63]"}
{ rank = same; "List[21]"; "Modifiers[54]"; "PrimitiveTypeAccess[56]"; "Modifiers[58]"; "PrimitiveTypeAccess[60]"; "List[64]"}
{ rank = same; "VarDeclStmt[22]"; "VarDeclStmt[30]"; "List[55]"; "List[59]"; "ReturnStmt[65]"}
{ rank = same; "Modifiers[23]"; "PrimitiveTypeAccess[25]"; "List[26]"; "Modifiers[31]"; "PrimitiveTypeAccess[33]"; "List[34]"; "Opt[66]"; "List[70]"; "UncheckedExceptions[66]"}
{ rank = same; "List[24]"; "VariableDeclarator[27]"; "List[32]"; "VariableDeclarator[35]"; "AddExpr[67]"}
{ rank = same; "List[28]"; "Opt[29]"; "List[36]"; "Opt[37]"; "VarAccess[68]"; "VarAccess[69]"}
{ rank = same; "MethodAccess[38]"}
{ rank = same; "List[40]"; "List[45]"; "List[46]"; "UncheckedExceptions[39]"}
{ rank = same; "AssignSimpleExpr[41]"; "VarAccess[44]"}
{ rank = same; "VarAccess[42]"; "IntegerLiteral[43]"}
"Program[0]" [label="Program\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[1]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"CompilationUnit[2]" [label="CompilationUnit\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[3]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[4]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ClassDecl[5]" [label="ClassDecl\n < VarDecl05 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[6]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[7]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifier[8]" [label="Modifier\n < public >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[9]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[10]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[11]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"MethodDecl[12]" [label="MethodDecl\n < m >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Entry[13]" [label="Entry\n ", fillcolor=white   style=filled  ]
"Modifiers[14]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[15]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[16]" [label="PrimitiveTypeAccess\n < void >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[17]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[18]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[19]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[20]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[21]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarDeclStmt[22]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[23]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[24]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[25]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[26]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[27]" [label="VariableDeclarator\n < int a >", fillcolor=white   style=filled  ]
"List[28]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[29]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarDeclStmt[30]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[31]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[32]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[33]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[34]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[35]" [label="VariableDeclarator\n < int x >", fillcolor=white   style=filled  ]
"List[36]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[37]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"MethodAccess[38]" [label="MethodAccess\n < add >", fillcolor=white   style=filled  ]
"List[40]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"AssignSimpleExpr[41]" [label="AssignSimpleExpr\n < a >", fillcolor=white   style=filled  ]
"VarAccess[42]" [label="VarAccess\n < a >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"IntegerLiteral[43]" [label="IntegerLiteral\n < 8 >", fillcolor=white   style=filled  ]
"VarAccess[44]" [label="VarAccess\n < a >", fillcolor=white   style=filled  ]
"List[45]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[46]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[39]" [label="UncheckedExceptions\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Exit[20]" [label="Exit\n ", fillcolor=white   style=filled  ]
"MethodDecl[47]" [label="MethodDecl\n < add >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Entry[48]" [label="Entry\n ", fillcolor=white   style=filled  ]
"Modifiers[49]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[50]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[51]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[52]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ParameterDeclaration[53]" [label="ParameterDeclaration\n < a >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[54]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[55]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[56]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ParameterDeclaration[57]" [label="ParameterDeclaration\n < b >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[58]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[59]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[60]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[61]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[62]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[63]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[64]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ReturnStmt[65]" [label="ReturnStmt\n ", fillcolor=white   style=filled  ]
"Opt[66]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"AddExpr[67]" [label="AddExpr\n < + >", fillcolor=white   style=filled  ]
"VarAccess[68]" [label="VarAccess\n < a >", fillcolor=white   style=filled  ]
"VarAccess[69]" [label="VarAccess\n < b >", fillcolor=white   style=filled  ]
"List[70]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[66]" [label="UncheckedExceptions\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Exit[63]" [label="Exit\n ", fillcolor=white   style=filled  ]
"CompilationUnit[2]" -> "List[3]" [style=dashed, color=gray]
"List[7]" -> "Modifier[8]" [style=dashed, color=gray]
"Modifiers[6]" -> "List[7]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Modifiers[6]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Opt[9]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[10]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Entry[13]" [style=dashed, color=gray]
"Modifiers[14]" -> "List[15]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Modifiers[14]" [style=dashed, color=gray]
"MethodDecl[12]" -> "PrimitiveTypeAccess[16]" [style=dashed, color=gray]
"MethodDecl[12]" -> "List[17]" [style=dashed, color=gray]
"MethodDecl[12]" -> "List[18]" [style=dashed, color=gray]
"Modifiers[23]" -> "List[24]" [style=dashed, color=gray]
"VarDeclStmt[22]" -> "Modifiers[23]" [style=dashed, color=gray]
"VarDeclStmt[22]" -> "PrimitiveTypeAccess[25]" [style=dashed, color=gray]
"VariableDeclarator[27]" -> "List[28]" [style=dashed, color=gray]
"VariableDeclarator[27]" -> "Opt[29]" [style=dashed, color=gray]
"List[26]" -> "VariableDeclarator[27]" [style=dashed, color=gray]
"VarDeclStmt[22]" -> "List[26]" [style=dashed, color=gray]
"List[21]" -> "VarDeclStmt[22]" [style=dashed, color=gray]
"Modifiers[31]" -> "List[32]" [style=dashed, color=gray]
"VarDeclStmt[30]" -> "Modifiers[31]" [style=dashed, color=gray]
"VarDeclStmt[30]" -> "PrimitiveTypeAccess[33]" [style=dashed, color=gray]
"VariableDeclarator[35]" -> "List[36]" [style=dashed, color=gray]
"AssignSimpleExpr[41]" -> "VarAccess[42]" [style=dashed, color=gray]
"AssignSimpleExpr[41]" -> "IntegerLiteral[43]" [style=dashed, color=gray]
"List[40]" -> "AssignSimpleExpr[41]" [style=dashed, color=gray]
"List[40]" -> "VarAccess[44]" [style=dashed, color=gray]
"MethodAccess[38]" -> "List[40]" [style=dashed, color=gray]
"MethodAccess[38]" -> "List[45]" [style=dashed, color=gray]
"MethodAccess[38]" -> "List[46]" [style=dashed, color=gray]
"MethodAccess[38]" -> "UncheckedExceptions[39]" [style=dashed, color=gray]
"Opt[37]" -> "MethodAccess[38]" [style=dashed, color=gray]
"VariableDeclarator[35]" -> "Opt[37]" [style=dashed, color=gray]
"List[34]" -> "VariableDeclarator[35]" [style=dashed, color=gray]
"VarDeclStmt[30]" -> "List[34]" [style=dashed, color=gray]
"List[21]" -> "VarDeclStmt[30]" [style=dashed, color=gray]
"Block[20]" -> "List[21]" [style=dashed, color=gray]
"Opt[19]" -> "Block[20]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Opt[19]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Exit[20]" [style=dashed, color=gray]
"List[11]" -> "MethodDecl[12]" [style=dashed, color=gray]
"MethodDecl[47]" -> "Entry[48]" [style=dashed, color=gray]
"Modifiers[49]" -> "List[50]" [style=dashed, color=gray]
"MethodDecl[47]" -> "Modifiers[49]" [style=dashed, color=gray]
"MethodDecl[47]" -> "PrimitiveTypeAccess[51]" [style=dashed, color=gray]
"Modifiers[54]" -> "List[55]" [style=dashed, color=gray]
"ParameterDeclaration[53]" -> "Modifiers[54]" [style=dashed, color=gray]
"ParameterDeclaration[53]" -> "PrimitiveTypeAccess[56]" [style=dashed, color=gray]
"List[52]" -> "ParameterDeclaration[53]" [style=dashed, color=gray]
"Modifiers[58]" -> "List[59]" [style=dashed, color=gray]
"ParameterDeclaration[57]" -> "Modifiers[58]" [style=dashed, color=gray]
"ParameterDeclaration[57]" -> "PrimitiveTypeAccess[60]" [style=dashed, color=gray]
"List[52]" -> "ParameterDeclaration[57]" [style=dashed, color=gray]
"MethodDecl[47]" -> "List[52]" [style=dashed, color=gray]
"MethodDecl[47]" -> "List[61]" [style=dashed, color=gray]
"AddExpr[67]" -> "VarAccess[68]" [style=dashed, color=gray]
"AddExpr[67]" -> "VarAccess[69]" [style=dashed, color=gray]
"Opt[66]" -> "AddExpr[67]" [style=dashed, color=gray]
"ReturnStmt[65]" -> "Opt[66]" [style=dashed, color=gray]
"ReturnStmt[65]" -> "List[70]" [style=dashed, color=gray]
"ReturnStmt[65]" -> "UncheckedExceptions[66]" [style=dashed, color=gray]
"List[64]" -> "ReturnStmt[65]" [style=dashed, color=gray]
"Block[63]" -> "List[64]" [style=dashed, color=gray]
"Opt[62]" -> "Block[63]" [style=dashed, color=gray]
"MethodDecl[47]" -> "Opt[62]" [style=dashed, color=gray]
"MethodDecl[47]" -> "Exit[63]" [style=dashed, color=gray]
"List[11]" -> "MethodDecl[47]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[11]" [style=dashed, color=gray]
"List[4]" -> "ClassDecl[5]" [style=dashed, color=gray]
"CompilationUnit[2]" -> "List[4]" [style=dashed, color=gray]
"List[1]" -> "CompilationUnit[2]" [style=dashed, color=gray]
"Program[0]" -> "List[1]" [style=dashed, color=gray]
"Exit[20]" -> "VariableDeclarator[35]" [color=red, constraint=false]
"VariableDeclarator[35]" -> "Exit[20]" [color=blue, constraint=false, xlabel=" "] 
"VariableDeclarator[35]" -> "MethodAccess[38]" [color=red, constraint=false]
"MethodAccess[38]" -> "VariableDeclarator[35]" [color=blue, constraint=false, xlabel=" "] 
"MethodAccess[38]" -> "VarAccess[44]" [color=red, constraint=false]
"VarAccess[44]" -> "MethodAccess[38]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[44]" -> "AssignSimpleExpr[41]" [color=red, constraint=false]
"AssignSimpleExpr[41]" -> "VarAccess[44]" [color=blue, constraint=false, xlabel=" "] 
"AssignSimpleExpr[41]" -> "IntegerLiteral[43]" [color=red, constraint=false]
"IntegerLiteral[43]" -> "AssignSimpleExpr[41]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[43]" -> "VariableDeclarator[27]" [color=red, constraint=false]
"VariableDeclarator[27]" -> "IntegerLiteral[43]" [color=blue, constraint=false, xlabel=" "] 
"VariableDeclarator[27]" -> "Entry[13]" [color=red, constraint=false]
"Entry[13]" -> "VariableDeclarator[27]" [color=blue, constraint=false, xlabel=" "] 
"Exit[63]" -> "ReturnStmt[65]" [color=red, constraint=false]
"ReturnStmt[65]" -> "Exit[63]" [color=blue, constraint=false, xlabel=" "] 
"ReturnStmt[65]" -> "AddExpr[67]" [color=red, constraint=false]
"AddExpr[67]" -> "ReturnStmt[65]" [color=blue, constraint=false, xlabel=" "] 
"AddExpr[67]" -> "VarAccess[69]" [color=red, constraint=false]
"VarAccess[69]" -> "AddExpr[67]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[69]" -> "VarAccess[68]" [color=red, constraint=false]
"VarAccess[68]" -> "VarAccess[69]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[68]" -> "Entry[48]" [color=red, constraint=false]
"Entry[48]" -> "VarAccess[68]" [color=blue, constraint=false, xlabel=" "] 
"List[3]" -> "List[4]" [style=invis]
"Modifiers[6]" -> "Opt[9]" -> "List[10]" -> "List[11]" [style=invis]
"MethodDecl[12]" -> "MethodDecl[47]" [style=invis]
"Entry[13]" -> "Modifiers[14]" -> "PrimitiveTypeAccess[16]" -> "List[17]" -> "List[18]" -> "Opt[19]" -> "Exit[20]" [style=invis]
"VarDeclStmt[22]" -> "VarDeclStmt[30]" [style=invis]
"Modifiers[23]" -> "PrimitiveTypeAccess[25]" -> "List[26]" [style=invis]
"List[28]" -> "Opt[29]" [style=invis]
"Modifiers[31]" -> "PrimitiveTypeAccess[33]" -> "List[34]" [style=invis]
"List[36]" -> "Opt[37]" [style=invis]
"AssignSimpleExpr[41]" -> "VarAccess[44]" [style=invis]
"VarAccess[42]" -> "IntegerLiteral[43]" [style=invis]
"Entry[48]" -> "Modifiers[49]" -> "PrimitiveTypeAccess[51]" -> "List[52]" -> "List[61]" -> "Opt[62]" -> "Exit[63]" [style=invis]
"ParameterDeclaration[53]" -> "ParameterDeclaration[57]" [style=invis]
"Modifiers[54]" -> "PrimitiveTypeAccess[56]" [style=invis]
"Modifiers[58]" -> "PrimitiveTypeAccess[60]" [style=invis]
"VarAccess[68]" -> "VarAccess[69]" [style=invis]

subgraph cluster_legend {
{node [style=filled, fillcolor=1, colorscheme="pastel13"]
Nodes [label="Total Nr. Nodes
74", fillcolor=1]
Edges [label="Total Nr. Edges
97", fillcolor=1]
NodesCFG [label="CFG Nr. Nodes
13", fillcolor=3]
EdgesCFG [label="CFG Nr. Edges
12", fillcolor=3]
Nodes-> "Program[0]" -> NodesCFG  [style="invis"]
Edges -> "Program[0]" ->EdgesCFG [style="invis"]}
label = "Statistics of the framework: NEXTFramework"
style="solid"
ranksep=0.05
nodesep=0.01
labelloc = b
len=2
}}