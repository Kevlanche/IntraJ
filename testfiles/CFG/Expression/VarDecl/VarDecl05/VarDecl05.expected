NodeName: Entry[13] Successors: [VariableDeclarator[27]]
NodeName: VariableDeclarator[27] Successors: [IntegerLiteral[43]]
NodeName: IntegerLiteral[43] Successors: [AssignSimpleExpr[41]]
NodeName: AssignSimpleExpr[41] Successors: [VarAccess[44]]
NodeName: VarAccess[44] Successors: [MethodAccess[38]]
NodeName: MethodAccess[38] Successors: [VariableDeclarator[35]]
NodeName: VariableDeclarator[35] Successors: [Exit[20]]
NodeName: Exit[20] Successors: []
NodeName: Entry[48] Successors: [VarAccess[68]]
NodeName: VarAccess[68] Successors: [VarAccess[69]]
NodeName: VarAccess[69] Successors: [AddExpr[67]]
NodeName: AddExpr[67] Successors: [ReturnStmt[65]]
NodeName: ReturnStmt[65] Successors: [Exit[63]]
NodeName: Exit[63] Successors: []

NodeName: Exit[20] Predecessor: [VariableDeclarator[35]]
NodeName: VariableDeclarator[35] Predecessor: [MethodAccess[38]]
NodeName: MethodAccess[38] Predecessor: [VarAccess[44]]
NodeName: VarAccess[44] Predecessor: [AssignSimpleExpr[41]]
NodeName: AssignSimpleExpr[41] Predecessor: [IntegerLiteral[43]]
NodeName: IntegerLiteral[43] Predecessor: [VariableDeclarator[27]]
NodeName: VariableDeclarator[27] Predecessor: [Entry[13]]
NodeName: Entry[13] Predecessor: []
NodeName: Exit[63] Predecessor: [ReturnStmt[65]]
NodeName: ReturnStmt[65] Predecessor: [AddExpr[67]]
NodeName: AddExpr[67] Predecessor: [VarAccess[69]]
NodeName: VarAccess[69] Predecessor: [VarAccess[68]]
NodeName: VarAccess[68] Predecessor: [Entry[48]]
NodeName: Entry[48] Predecessor: []
